javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/OpenTelemetryInstaller.java

```diff
@@ -20,6 +22,7 @@
   private static final Logger logger = LoggerFactory.getLogger(OpenTelemetryInstaller.class);
 
   static final String JAVAAGENT_ENABLED_CONFIG = "otel.javaagent.enabled";
+  static final String JAVAAGENT_NOOP_CONFIG = "otel.javaagent.experimental.sdk.noop";
```

Comment:
How about
```suggestion
  static final String JAVAAGENT_NOOP_CONFIG = "otel.javaagent.experimental.use-noop-api";
```
? It's noop API, not SDK (see the module name), so we can reflect that in the property name.